src/mscorlib/src/System/ReadOnlySpan.cs

```diff
@@ -328,7 +328,7 @@ public T[] ToArray()
         public static ReadOnlySpan<T> Empty => default(ReadOnlySpan<T>);
     }
 
-    public static class ReadOnlySpanExtensions
+    public static class ReadOnlySpanExtensionMethods
```

Comment:
This does not feel right. I think we need a better plan for factoring the Span extension methods than this as I have mentioned over email.